### PR TITLE
Update attendee-demographics.md

### PR DESCRIPTION
**Pull request by germonprez**
Signed-off-by: Matt Germonprez <germonprez@gmail.com>

Making some updates to the Attendee Demographics metric to better reflect a metric focused on demographics.

### CODE REVIEW

**Comment by Nebrethar** [comment]
I like these changes. They make sense to me.

File: focus-areas/events/attendee-demographics.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Attendees Demographics
+# Attendee Demographics
 
-Question: How diverse and inclusive are the attendees?
+Question: How diverse and inclusive are the attendees of an event?
```
Comment (ElizabethN) [inline focus-areas/events/attendee-demographics.md:3]: Might suggest rewording this, as an attendee itself isn't diverse or inclusive, but we are referencing the body of attendees and the policies of the event.

Comment (Nebrethar) [reply focus-areas/events/attendee-demographics.md:3]: "How diverse and inclusive is an event with regard to attendees"?
